admin/admin.go

```diff
@@ -13,45 +15,53 @@ import (
 
 // Server manages InfluxDB's admin web server.
 type Server struct {
-	port     string
+	sync.Mutex
```

Comment:
Are you deliberating embedding `sync.Mutex`? We have not done this in other parts of course, AFAIK.